cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,28 @@
+class TimeLoggingPage {
+
+    assertPageTimeLogginDisplayed() {
+        return cy.url().should('include', '/time-logging');
+    }
+
+    getUserProfileButtonSelector() {
+        return cy.get('.user-info__title');
+    }
+
+    getNavListSelector() {
```

Comment:
Here also method naming could be better, as you return an element not a selector, also navigation list is not tied to specific page, so it would not be logical to use "timeLoggingPage" if you want to check navigation list while in task page. I suggest putting navigation list methods to different file like `pageComponents.js` where you would put all the stuff that is shared between pages

Reply:
fixed with commit 'f073c5'